Documentation/bpf.rst

```diff
@@ -1181,6 +1181,64 @@ The generated LLVM IR can also be dumped in human readable format through:
 
     $ clang -O2 -Wall -emit-llvm -S -c xdp-example.c -o -
 
+In addition, For conversion debugging information to BTF (BPF Type Format),
+more recent ``llc`` and ``pahole`` are needed and ``llc`` and ``pahole``
+can be probed for BTF support:
+
+::
+
+    $ llc -march=bpf -mattr=help |& grep dwarfris
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
```

Comment:
I think this would need a short explanation what `dwarfris` is about and why it is needed for BPF to clarify it better to the reader.

Reply:
OK, I'll add it  leveraging the below description why `dwarfris` is needed for BTF.